### ISSUE TITLE
`ReliableDeliverySpec` has repeatedly failed in the Scala community build

### ISSUE THREAD

**Issue by SethTisue** [akka/akka#29340]
seen on JDK 8 and 11 and 14

**akka-more** fail Jun 24
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/989/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail May 16
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/919/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail May 9
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/900/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail May 8
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/899/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail May 2
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/891/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail Apr 23
  https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-community-build/3426/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail Apr 6
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/826/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail Apr 4
  https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-community-build/3358/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail Mar 30
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/802/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec
**akka-more** fail Mar 20
  https://scala-ci.typesafe.com/job/scala-2.13.x-jdk11-integrate-community-build/1262/artifact/logs/akka-more-build.log
  akka.actor.typed.delivery.ReliableDeliverySpec

**Comment by patriknw** [comment]
@SethTisue Which Akka version (commit) is this?

**Comment by SethTisue** [comment]
the commit used is the first line in each log. so e.g. for the June 24 [failure](https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/989/artifact/logs/akka-more-build.log) we see:

```
[akka-more] Resolving: https://github.com/akka/akka.git#fa9ceaa5eb3c12ebc1e2ac5a3c121b9d19ef7fc1 in directory: ./target-0.9.16/project-builds/akka-more-8ce90334ce0439e399622e9b8812cd9a8261569d
```

**Comment by SethTisue** [comment]
That commit is from March 2020. The comment near the top of https://github.com/scala/community-build/blob/2.13.x/proj/akka.conf shows we are still using an old commit because of the akka -> akka-persistence-cassandra -> lagom dependency chain; we are waiting for lagom to move to akka-persistence-cassandra 1.0

**Comment by patriknw** [comment]
> the commit used is the first line

Thanks. I'll take a look to see if we fixed anything or if the test is sensitive to some timing (or if it's a bug).

**Comment by patriknw** [comment]
I couldn't reproduce it locally with that commit. Rather much has changed compared to master so I don't want to spend time debugging this unless it reoccurs with latest master.

> we are waiting for lagom to move to akka-persistence-cassandra 1.0

That is not planned to happen any time soon. Lagom should work with latest Akka version though.

**Comment by SethTisue** [comment]
Okay, I'll try latest Akka again.

**Comment by SethTisue** [comment]
related: will Play be moving to akka-http 10.2.0 soon, or should I proceed on the assumption that the community build needs to stick with 10.1.x for now?

**Comment by patriknw** [comment]
> will Play be moving to akka-http 10.2.0 soon

That is a bigger task and requires new Play 2.9.0 version. Nothing planned.

**Comment by SethTisue** [comment]
note to self: if the akka-persistence-cassandra thing can't be resolved, perhaps a way out (other than dropping lagom, which I don't want to do) would be to use an akka-persistence-cassandra binary from Maven Central instead of building it from source

**Comment by SethTisue** [comment]
it failed today: https://scala-ci.typesafe.com/job/scala-2.13.x-jdk14-integrate-community-build/1177/artifact/logs/akka-more-build.log

**Comment by patriknw** [comment]
Good, then I have something to work on.

**Comment by patriknw** [comment]
@SethTisue this should be fixed in master now (51045a295b690a588a3fb69e931aea9f6f5f7a3f)
